example/README.md

```diff
@@ -29,7 +30,8 @@ kind load docker-image --name topolvm-example ghcr.io/topolvm/pvc-autoresizer:de
 Deploy `pvc-autoresizer`:
 
 ```
-make deploy
+helm repo add pvc-autoresizer https://topolvm.github.io/pvc-autoresizer/
+helm install --create-namespace --namespace pvc-autoresizer pvc-autoresizer pvc-autoresizer/pvc-autoresizer --set "controller.args.prometheusURL=http://prometheus-kube-prometheus-prometheus.prometheus.svc:9090"
```

Comment:
It would be better to order the arguments should be the same as in `helm install` in "Setup Prometheus".
```suggestion
helm install pvc-autoresizer pvc-autoresizer/pvc-autoresizer --create-namespace --namespace=pvc-autoresizer --set "controller.args.prometheusURL=http://prometheus-kube-prometheus-prometheus.prometheus.svc:9090"
```